src/ContinuumWorld.jl

```diff
@@ -52,11 +54,11 @@ const default_rewards = [-10.0, -5.0, 10.0, 3.0]
 end
 
 POMDPs.actions(w::CWorld) = w.actions
-POMDPs.n_actions(w::CWorld) = length(w.actions)
 POMDPs.discount(w::CWorld) = w.discount
 
-function POMDPs.gen(::DDNNode{:sp}, w::CWorld, s::AbstractVector, a::AbstractVector, rng::AbstractRNG)
-    return s + a + w.stdev*randn(rng, Vec2)
+function POMDPs.gen(w::CWorld, s::AbstractVector, a::AbstractVector, rng::AbstractRNG)
+    sp = s + a + w.stdev*randn(rng, Vec2)
+    return (sp=sp,)
 end
```

Comment:
When implementing only the state transition, it is better to use `POMDPModelTools.ImplicitDistribution` than `gen`. `gen` is for when you need to return multiple things (reward, observation, state) from one function.

```suggestion
function POMDPs.transition(w::CWorld, s::AbstractVector, a::AbstractVector)
    ImplicitDistribution(w, s, a) do w, s, a, rng
        return s + a + w.stdev*randn(rng, Vec2)
    end
end
```